packages/shared/src/components/post/PostModalActions.tsx

```diff
@@ -56,6 +62,14 @@ export function PostModalActions({
     });
   };
 
+  const onReadArticle = () => {
+    trackEvent(
+      postAnalyticsEvent('click', post, {
```

Comment:
Let's change the event name so we will not be confused with the click on the card.
I know we used to have `click` before on the article page but now it's a great time to change it.
Also is it used by the article page as well?

```suggestion
      postAnalyticsEvent('go to link', post, {
```

Reply:
Yes. This button will be available on the article page as well.